### DISCUSSION TITLE
Welcome to paystack-gateway Discussions!

### DISCUSSION THREAD

**Discussion by ezkemboi** [momi-foundation-coding/paystack-gateway#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0Kiprotich** [comment]
Session begins when

**Comment by ezkemboi** [comment]
Today, or any day.
I am trying to build a wrapper for the same.
For more information, check the documentation on
- https://paystack.com/docs/api/

**Comment by ezkemboi** [comment]
We can also start creating issues, but, I can give more clarification on the same.

**Comment by KipsangEmmanuel** [comment]
bro tuma wifi password tuko na nick

**Reply by KipsangEmmanuel** [reply]
@ezkemboi
